ui.apps/src/main/content/META-INF/vault/filter.xml

```diff
@@ -1,7 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <workspaceFilter version="1.0">
+    <filter root="/apps/cq/core/content/nav/tools/aem-custom-lists"/>
+    <filter root="/apps/wcm/core/content/sites/actions/secondary/create/items/createlist"/>
+
+    <filter root="/apps/authoring-toolkit/clientlib-injector"/>
+    <filter root="/apps/authoring-toolkit/custom-lists"/>
     <filter root="/apps/authoring-toolkit/depends-on"/>
-    <filter root="/apps/authoring-toolkit/depends-on-injector"/>
     <filter root="/apps/authoring-toolkit/install"/>
+
```

Comment:
Mb, add the comment regarding retirement purposes of the following paths.

Reply:
done